message-index/messages/GHC-00482/example1/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Lambda syntax in `case` expression
+---
+
+When pattern matching with a `case` expression, backslash (`\`) is not required before the pattern.
```

Comment:
```suggestion
When pattern matching with a `case` expression, anonymous functions (that is, functions that begin with`\`) are not allowed before the pattern, because functions cannot be pattern-matched on.
```